lib/databases/mongodb.js

```diff
@@ -290,9 +290,11 @@ _.extend(Mongo.prototype, {
   ensureIndexes: function() {
     var self = this;
 
-    if (!this.isConnected || !this.collectionName || !this.indexes) return;
+    var indexes = (this.repositorySettings &&  this.repositorySettings.mongodb && this.repositorySettings.indexes) ? this.repositorySettings.indexes : this.indexes 
```

Comment:
shouldn't this be:

`var indexes = (this.repositorySettings &&  this.repositorySettings.mongodb && this.repositorySettings.mongodb.indexes) ? this.repositorySettings.mongodb.indexes : this.indexes;`